.gitlab-ci.yml

```diff
@@ -862,10 +862,10 @@ plugin:ci-elpi:
   extends: .ci-template
 
 plugin:ci-equations:
-  extends: .ci-template
+  extends: .ci-template-flambda
```

Comment:
I haven't followed why this change is made, but if you do it, you also need to change the reverse dependencies (in this case it's metacoq).

Reply:
To avoid compiling equations with native-compiler sinc eit fails, thanks.

Reply:
like @Zimmi48, I don't understand why you replaced `.ci-template` with `.ci-template-lambda`, this seems orthogonal with disabling native?

cf. this code excerpt:

https://github.com/coq/coq/blob/3207bab23a5ffc649b4c33c49b43112747a1781f/.gitlab-ci.yml#L256-L264